app/mod_auth/__init__.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# -- encoding: utf-8 --
+#
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-Core project.
+#
```

Comment:
This file is described as part of "FI-Core", while preceding are part of "FI-WARE". Please unify these headers.